rlptxn/constraints.lisp

```diff
@@ -829,10 +829,9 @@
 
 ;; 4.5.2.15
 (defconstraint phaseAccessList-updateAddrLookUp (:guard IS_PHASE_ACCESS_LIST)
-  (if-zero (+ [DEPTH 2]
-              (- (prev nADDR) nADDR))
-           (begin (remained-constant! ADDR_HI)
-                  (remained-constant! ADDR_LO))))
+  (if-not-zero (* [DEPTH 1]  (- nADDR (- (prev nADDR) 1)))
+                     (begin (remained-constant! ADDR_HI)
+                            (remained-constant! ADDR_LO))))
```

Comment:
I'm not following how this maps to

<img width="811" alt="image" src="https://github.com/user-attachments/assets/1497f5db-d124-4d8d-b187-2610bee4e3ae">

Reply:
I guess this relates to this spec update: https://github.com/Consensys/linea-specification-internal/pull/618

Reply:
My next question would be: are special precautions required for the first row of `IS_PHASE_ACCESS_LIST = 1` ?

Reply:
No, the first row of ACCESS£LIST PHASE  has DEPTH_1 = 0, so it's OK.

Reply:
see 
![Uploading image.png…]()